services/app-api/prisma/schema.prisma

```diff
@@ -166,6 +165,17 @@ model RateRevisionTable {
   contractsWithSharedRateRevision ContractTable[]          @relation(name: "SharedRateRevisions")
 }
 
+model ContractActionTable {
```

Comment:
@macrael @JasonLin0991 how do you feel about this implementation, given that after speaking with Kelsey this morning, it sounds like "unapproval" and the idea of having multiple approval actions is something confined to V1. In V2 when we implement sending the approval letter, approval will be considered final and there won't be multiples

Reply:
> In V2 when we implement sending the approval letter, approval will be considered final and there won't be multiples (with the caveat that final confirmation hasn't been set yet but hopefully will have a final idea on this soon)

This should be okay for v1. Do you expect the schema to change in v2?

Reply:
Sorry @JasonLin0991 I didn't see your comment before I posted mine. I don't expect the schema to change in V2.